### ISSUE TITLE
defaultRate = defaultDevice.get('default_samplerate')

### ISSUE THREAD

**Issue by lukaszliniewicz** [erdewit/HiFiScan#19]
Thank you for handling the previous issue so quickly! Unfortunately, I get another error after upgrading:

```
C:\Users\Pc>python -m hifiscan
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\__main__.py", line 4, in <module>
    main()
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 685, in main
    app = App()
          ^^^^^
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 46, in __init__
    self.resetAudio()
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 97, in resetAudio
    defaultRate = defaultDevice.get('default_samplerate')
                  ^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
```

_Originally posted by @lukaszliniewicz in https://github.com/erdewit/HiFiScan/issues/18#issuecomment-1831893576_

**Comment by erdewit** [comment]
Ah yes of course. Thanks again, this should now be really fixed in v1.5.2.

**Comment by lukaszliniewicz** [comment]
Thank you, it works now! :)
